trappy/systrace.py

```diff
@@ -14,6 +14,7 @@
 #
 
 from trappy.ftrace import GenericFTrace
+import re
 
 class drop_before_trace(object):
```

Comment:
we have been naming the classes as DropBeforeTrace. Also I am not sure what the name conveys. I will change that in a follow up PR.